geomesa-compute/pom.xml

```diff
@@ -52,6 +52,13 @@
             <version>14.0.1</version>
             <scope>provided</scope>
         </dependency>
+        <!-- spark compatible version of apache curator -->
+        <dependency>
+            <groupId>org.apache.curator</groupId>
+            <artifactId>curator-recipes</artifactId>
+            <version>2.4.0</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
We probably need a CQ for making it runtime in geomesa-accumulo. Previously it was scope 'test' in geomesa-kafka and scope 'provided' in geomesa-compute.